README.md

```diff
@@ -1,2 +1 @@
-# Quick start
-`make vendor`
\ No newline at end of file
+Look for the README inside the ledger-core/v2 folder.
```

Comment:
It is going to change in the next sprint, I think. https://insolar.atlassian.net/browse/PLAT-155